### DISCUSSION TITLE
Problems with new blocks in the project 😔

### DISCUSSION THREAD

**Discussion by emrahkan** [jeffreyvr/dropblockeditor#30] [General]
Thank you for the great project! I have been looking for a possibility for a long time. However, I don't know what I've done wrong. As soon as I change something and add a new block, I get the error message "null". When I add several blocks, some blocks suddenly disappear. Do you have any idea what could be causing this?

https://github.com/jeffreyvr/dropblockeditor/assets/17336614/92ab3ccd-ba22-4b0e-88fe-b79177b5bac4

**Comment by jeffreyvr** [comment]
Can you check if the issue is resolved with version 0.3.1?

**Reply by emrahkan** [reply]
wow, thanks for the quick bugfixing 🥳
